src/components/Shared/Filters/TagsSection/FilterTagDeleteModal.tsx

```diff
@@ -36,11 +34,7 @@ export const FilterTagDeleteModal: React.FC<FilterTagDeleteModalProps> = ({
         page,
       },
       refetchQueries: [
-        {
-          query:
-            page === 'contacts' ? ContactFiltersDocument : TaskFiltersDocument,
-          variables: { accountListId },
-        },
+        page === 'contacts' ? ContactFiltersDocument : TaskFiltersDocument,
```

Comment:
For some reason, how this was wasn't causing the data to refresh. Removing the variables makes it work though.

Reply:
I wonder if it saw it as a different query